### PR TITLE
Fix company counting

### PR DESCRIPTION
**Pull request by caleybrock**

### CODE REVIEW

**Comment by tanyaparker** [comment]
The changes look good but I haven't tested it yet. Can we get someone else to also test while we work on the UI changes?

**Comment by caleybrock** [comment]
@breville can you take a look?

File: pegasus/sites.v3/code.org/public/playlab.haml [hunk 1]
```diff
@@ -23,20 +23,28 @@ social:
...
 %div{style: "clear:both"}
 
+-company = request.GET['company']
+
 .playlab-container
   %h1.header= I18n.t(:playlab_choose_theme)
   %div.image-collection
-    %a.white-link{href: "/api/hour/begin/playlab"}
+    - url = "/api/hour/begin/playlab"
+    - url = url + "?company=#{company}" if company
```
Comment (philbogle) [inline pegasus/sites.v3/code.org/public/playlab.haml:32]: Company needs to be URL encoded here;  e.g. AT&T would be break.

File: pegasus/sites.v3/code.org/public/playlab.haml [hunk 2]
```diff
@@ -23,20 +23,28 @@ social:
...
+    %a.white-link{href: url}
       %div.choice
         %div.choice-container
           %div.choice-label.purple= I18n.t(:playlab_classic_title)
           %img.character-img{src: "/images/playlab_original3.png"}
-    %a.white-link{href: "/api/hour/begin/gumball"}
+    - url = "/api/hour/begin/gumball"
+    - url = url + "?company=#{company}" if company
```
Comment (philbogle) [inline pegasus/sites.v3/code.org/public/playlab.haml:39]: URL encoding also here an elsewhere

File: pegasus/sites.v3/code.org/public/playlab.haml [hunk 3]
```diff
@@ -23,20 +23,28 @@ social:
...
+    %a.white-link{href: url}
       %div.choice
         %div.choice-container
           %div.choice-label.blue= I18n.t(:playlab_gumball_title)
           %img.character-img{src: "/images/playlab_gumball.png"}
-    %a.white-link{href: "/api/hour/begin/iceage"}
+    - url = "/api/hour/begin/iceage"
```
Comment (philbogle) [inline pegasus/sites.v3/code.org/public/playlab.haml:45]: After HOC, consider using the company cookie to remember the company.

Alternatively, to reduce duplicated code, please add a helper to build the /api/hour/begin URL

**Pull request closed without merging**